frameworks/cassandra/dashboards/README.md

```diff
@@ -0,0 +1,16 @@
+# Cassandra dashboards for DC/OS
+
+This is a folder for Grafana dashboards. In order to install a dashboard on a cluster, `dcos-monitoring` package need to be installed (as well as `cassandra` package itself:
+
+1. Install required package: `dcos package install beta-dcos-monitoring`
```

Comment:
is this true anymore? I think with GA users can now install `dcos-monitoring` (without the `beta-`). 

Can you confirm these instructions work completely?

Reply:
dcos-monitoring not yet released, its in SOAK.

https://mesosphere.slack.com/archives/CCWLY6SLE/p1558627650002000

Reply:
okay thanks for confirming - then this PR is good to go 👍